### PR TITLE
Styling (Issue #15): Make the shopping list look professional and welcoming.

### LINKED ISSUE #15: 15. As a user, I want the shopping list screen to look pretty so the app looks professional and welcoming.
AC:

* The shopping list screen (the screen that lists all of the user’s purchases) looks something like the following (or equivalent, use your creativity!):

shopping-list

### PR DESCRIPTION
**Pull request by CodeMeKathy**
## Description

This Pull Request (PR) adds styling to the `<List />` to aid the app in looking more professional and welcoming.

## Related Issue

Closes Issue #15

## Acceptance Criteria

- [x] The shopping list screen (the screen that lists all of the user’s purchases) looks something like the following (or equivalent, use your creativity!):
image

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|     | :sparkles: New feature     |
| ✓   | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before
Before - Shopping List

### After
After - Shopping List

## Testing Steps / QA Criteria

- From your terminal, run `git checkout vh-kl-pretty-shopping-list` in the `tcl-11-smart-shopping-list ` repository.
- Once on the `vh-kl-pretty-shopping-list` branch, run npm start to load the server and start the browser.
- Visit the `<Home />` component at http://localhost:3000.
- Create a new shopping list by clicking the 'Create a New Shopping List' button or select an existing shopping list to be redirected to the `<List />` component view.

### CODE REVIEW

**Review by brewswain — APPROVED** [review]
Man, this is really gorgeous, I'm a huge fan of the direction yall took it, and the decision to use a Card style display for the shopping list was really inspired!

File: src/services/Listener/Listener.service.jsx [hunk 1]
```diff
@@ -106,12 +110,27 @@ const Listener = props => {
     <>
       <div className="listener__container">{isEmpty ? <Card /> : null}</div>
       <div className="search__bar">
+        {/* Original Input
```
Comment (joeylaguna) [inline src/services/Listener/Listener.service.jsx]: Can line 113-120 be removed?

File: src/services/Listener/Listener.service.jsx [hunk 2]
```diff
@@ -141,8 +160,28 @@ const Listener = props => {
           ))
         ) : (
           <div>
-            {filteredItems.map(filteredItem => (
+            {/* {filteredItems.map(filteredItem => (
```
Comment (joeylaguna) [inline src/services/Listener/Listener.service.jsx]: Can this be removed?

File: src/services/Listener/Listener.service.jsx [hunk 3]
```diff
@@ -141,8 +160,28 @@ const Listener = props => {
...
               <div> {filteredItem} </div>
+            ))} */}
+            {filteredItems.map(item => (
+              <div>
+                {' '}
+                {item}
+                <Item
+                  key={item.id}
+                  name={item.name}
+                  // id={item.id}
```
Comment (joeylaguna) [inline src/services/Listener/Listener.service.jsx]: Do we need this?

File: src/components/List/List.style.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
+@import url('https://fonts.googleapis.com/css2?family=Dancing+Script&display=swap');
+
+.page__title {
+  font-family: 'Dancing Script', cursive;
+  font-size: 3rem;
+}
+
+// Media Queries
+@media screen and (max-width: 425px) {
```
Comment (joeylaguna) [inline src/components/List/List.style.scss:9]: Love seeming media queries!

**Comment by stef-codes** [comment]
Love the styling! Some things I noticed: 

- Responsiveness: When I start to add more items and I don't expand to full screen the items don't stay on the card

Screen Shot 2020-09-27 at 10 30 55 AM

- Routing:. When you choose an existing list you're directed to the list route and component. But, you can't return back to the list route without starting the process again from home. This is has been a problem before.

After Choosing a Token: 
Screen Shot 2020-09-27 at 10 27 31 AM
After Choosing a Token and adding an item: 
Screen Shot 2020-09-27 at 10 44 07 AM

- Items Accordion: Only the details for the first item show. The rest don't expand in a way to see the details. 
Screen Shot 2020-09-27 at 10 50 31 AM

**Review by stef-codes — COMMENTED** [review]
My review is in the previous comment. Great work!

**Comment by daredtech** [comment]
> My review is in the previous comment. Great work!

Hi @stef-codes, I can not reproduce the issue. Selecting token and adding items works when I do it. 
Before:
Screen Shot 2020-09-27 at 11 26 07 AM
After:
Screen Shot 2020-09-27 at 11 26 32 AM

**Comment by daredtech** [comment]
> Routing:. When you choose an existing list you're directed to the list route and component. But, you can't return back to the list route without starting the process again from home. This is has been a problem before.

@stef-codes 
I am not sure what you mean,  your list (or `your list is empty` card) is always present on a screen. See the screenshots above. The only reason you would want to go to `Home` is to select a new list/create a new list. You can't go to the list without specifying to which list you are going (which is done by selecting token).

**Comment by daredtech** [comment]
> * Responsiveness: When I start to add more items and I don't expand to full screen the items don't stay on the card
> 

@stef-codes Great catch. Thank you!

**Review by joeylaguna — APPROVED** [review]

File: src/services/Listener/Listener.service.jsx (hunk 3 above)
Comment (CodeMeKathy) [reply src/services/Listener/Listener.service.jsx]: Thank you for the reminder to remove unneeded code and comments.

**Pull request merged**